fix hanging during building of documentation (hanging sphinx-build processes),
and prevent auto-downloading of Sphinx and dependencies to build documentation (already provided via EasyBuild)
author: Kenneth Hoste (HPC-UGent)
--- lammps-stable_3Mar2020/cmake/Modules/Documentation.cmake.orig	2020-03-03 16:27:12.000000000 +0100
+++ lammps-stable_3Mar2020/cmake/Modules/Documentation.cmake	2020-05-03 11:26:14.441071541 +0200
@@ -3,8 +3,6 @@
 ###############################################################################
 option(BUILD_DOC "Build LAMMPS documentation" OFF)
 if(BUILD_DOC)
-  include(ProcessorCount)
-  ProcessorCount(NPROCS)
   find_package(PythonInterp 3 REQUIRED)
 
   set(VIRTUALENV ${PYTHON_EXECUTABLE} -m virtualenv)
@@ -22,14 +20,14 @@
     OUTPUT requirements.txt
     DEPENDS docenv
     COMMAND ${CMAKE_COMMAND} -E copy ${LAMMPS_DOC_DIR}/utils/requirements.txt requirements.txt
-    COMMAND ${DOCENV_BINARY_DIR}/pip install -r requirements.txt --upgrade
+    #COMMAND ${DOCENV_BINARY_DIR}/pip install -r requirements.txt --upgrade
     COMMAND ${DOCENV_BINARY_DIR}/pip install --upgrade ${LAMMPS_DOC_DIR}/utils/converters
   )
 
   add_custom_command(
     OUTPUT html
     DEPENDS ${DOC_SOURCES} docenv requirements.txt
-    COMMAND ${DOCENV_BINARY_DIR}/sphinx-build -j ${NPROCS} -b html -c ${LAMMPS_DOC_DIR}/utils/sphinx-config -d ${CMAKE_BINARY_DIR}/doctrees ${LAMMPS_DOC_DIR}/src html
+    COMMAND sphinx-build -b html -c ${LAMMPS_DOC_DIR}/utils/sphinx-config -d ${CMAKE_BINARY_DIR}/doctrees ${LAMMPS_DOC_DIR}/src html
   )
 
   add_custom_target(
